simapp/integration/client/grpc/tmservice/service_test.go

```diff
@@ -31,7 +31,7 @@ type IntegrationTestSuite struct {
 }
 
 func TestIntegrationTestSuite(t *testing.T) {
-	//t.Skip() // to be re-enabled in https://github.com/cosmos/cosmos-sdk/pull/12482/
+	// t.Skip() // to be re-enabled in https://github.com/cosmos/cosmos-sdk/pull/12482/
```

Comment:
```suggestion
```

This line can now be removed, see https://github.com/cosmos/cosmos-sdk/pull/12482#issuecomment-1193186276